corehq/apps/accounting/tests/test_stripe_payment.py

```diff
@@ -35,6 +35,19 @@ def test_when_stripe_errors_no_payment_record_exists(self, mock_create):
 
         self.assertEqual(PaymentRecord.objects.count(), 0)
 
+    @patch.object(stripe.Charge, 'create')
+    @patch.object(PaymentRecord, 'create_record')
+    def test_when_create_record_fails_stripe_is_not_charged(self, mock_create_record, mock_create):
+        mock_create_record.side_effect = Exception
+
+        try:
+            self._call_process_request()
```

Comment:
would've done that but the exception gets caught in ```process_request``` with the change I implemented. I was more interested in exposing the bad side-effect where our records get out of sync with what's been charged to stripe